### ISSUE TITLE
elan: build fails with "note: ld: library not found for -liconv"

### ISSUE THREAD

**Issue by samuela** [NixOS/nixpkgs#124217]
**Describe the bug**
On x86_64-darwin, the `elan` package is currently broken. In particular it ends with a failure
```
  = note: "/nix/store/2pb1r2zgw6yiabv4v3d4j66d3vwgdd5q-clang-wrapper-7.1.0/bin/cc" "-m64" "-arch" "x86_64" "-L" "/nix/store/akm781c35gr7mb14g5h0kqlk3cvyzmij-rustc-1.51.0/lib/rustlib/x86_64-apple-darwin/lib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/elan_init.elan_init.d0pdepo8-cgu.0.rcgu.o" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/elan_init.elan_init.d0pdepo8-cgu.1.rcgu.o" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/elan_init.elan_init.d0pdepo8-cgu.10.rcgu.o" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/elan_init.elan_init.d0pdepo8-cgu.11.rcgu.o" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/elan_init.elan_init.d0pdepo8-cgu.12.rcgu.o" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/elan_init.elan_init.d0pdepo8-cgu.13.rcgu.o" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/elan_init.elan_init.d0pdepo8-cgu.14.rcgu.o" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/elan_init.elan_init.d0pdepo8-cgu.15.rcgu.o" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/elan_init.elan_init.d0pdepo8-cgu.2.rcgu.o" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/elan_init.elan_init.d0pdepo8-cgu.3.rcgu.o" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/elan_init.elan_init.d0pdepo8-cgu.4.rcgu.o" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/elan_init.elan_init.d0pdepo8-cgu.5.rcgu.o" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/elan_init.elan_init.d0pdepo8-cgu.6.rcgu.o" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/elan_init.elan_init.d0pdepo8-cgu.7.rcgu.o" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/elan_init.elan_init.d0pdepo8-cgu.8.rcgu.o" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/elan_init.elan_init.d0pdepo8-cgu.9.rcgu.o" "-o" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/elan_init" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/elan_init.17pvzqwrfa2tefzc.rcgu.o" "-Wl,-dead_strip" "-nodefaultlibs" "-L" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps" "-L" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/release/deps" "-L" "/nix/store/wmdkisbdf4ih0yla1kzaj769ri372vrw-openssl-1.1.1k/lib" "-L" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/build/bzip2-sys-88cdf84f980808cd/out/lib" "-L" "/nix/store/akm781c35gr7mb14g5h0kqlk3cvyzmij-rustc-1.51.0/lib/rustlib/x86_64-apple-darwin/lib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/libwait_timeout-98faeec9177fa491.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/libmarkdown-a1f6f78ba13abeba.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/liblazy_static-91fab57dd04714a5.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/libpipeline-0defa3e3ae5ddcd7.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/libtempdir-4e89bfea657da07e.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/librand-fd40f01bba973b1d.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/libterm-bb4c1010f4748de6.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/libdirs_next-45faeb34d97446d5.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/libdirs_sys_next-310b7773adecc9ba.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/libelan-d903b57fbd22c006.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/libtime-7a6a2c590358fd4f.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/libtime_macros-e00ac4567c9c4c28.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/libstandback-cab61b1058c8533e.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/libtempfile-3eba8b4d2df78f54.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/libremove_dir_all-c869bece10c191d0.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/libserde_json-9317c4754dd4b234.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/libryu-db8bc1340f9d7ae8.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/libitoa-b445b0b9ceb6ffec.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/libclap-f24ddd11a20ccda5.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/libvec_map-ebc2401d6ce65bf6.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/libtextwrap-468bc1e9f1fb5a63.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/libunicode_width-ed88f1a110456c24.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/libstrsim-e403783120974290.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/libbitflags-0d253ad7730923dc.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/libatty-2f39b0759e24500d.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/libansi_term-5cd777ddfa814a5f.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/libelan_dist-e0c36a041994a169.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/libzip-25cfff0b1edf5a19.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/libtime-347706e0633e12d8.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/libthiserror-2c67bc5bba1ac4a5.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/libbzip2-82e475d615df8658.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/libbzip2_sys-55c85037afa92295.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/libbyteorder-5600a45258cb5a94.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/libjson-169d01c702e5785e.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/libelan_utils-4bfbdef9bec63b59.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/libremove_dir_all-7d42da3a277389be.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/libdirs-ad1692302be6fc0d.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/libdirs_sys-e3c417f2dc70a064.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/libsemver-1156466a89715718.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/libsemver_parser-cfd8b82c89eb3592.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/libpest-3642229cc5e1d51c.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/libucd_trie-533b1fcfca888b81.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/libdownload-6f6befb9de4fa2a9.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/libcurl-97f013412df2c00c.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/libsocket2-aafa43800eb3f665.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/libcurl_sys-ecde3f257ea09c19.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/libsha2-dca0648a37339bc4.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/libcpuid_bool-edf389e4a5a517a8.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/libopaque_debug-481ba35a442effb9.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/libdigest-8bebed98b99bdaa3.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/libblock_buffer-0541a75bc75b5f90.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/libgeneric_array-081a368017c23f63.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/libtypenum-ff42404c8799a28f.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/liberror_chain-2a180199c7117b8a.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/libbacktrace-2fe8ef79b5f8cd74.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/libobject-3900306feedb4b8a.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/libaddr2line-96597418f1d25d5b.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/libgimli-c38ac1323c187e61.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/librustc_demangle-92b08f5badeb00ce.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/libscopeguard-7123071295e2bf62.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/librand-fd396db452ddfec4.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/librand_chacha-7cf1e87d57e4e37c.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/libppv_lite86-fb1758143cdf3e52.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/librand_core-eeb3595d99572e15.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/libgetrandom-b7b4253013a3d838.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/liburl-75040b1bc39ea59d.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/libidna-3e603bdee75e6f89.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/libunicode_normalization-d103b24536b2e403.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/libtinyvec-afff300ee1627e5a.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/libtinyvec_macros-4ddbd7b9e616624d.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/libunicode_bidi-680980a85415a93f.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/libform_urlencoded-22053f5873be4cfb.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/libpercent_encoding-1ad397d61d036dd2.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/libmatches-cea510a368e42bee.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/libtar-26aa4b3c423d3796.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/libxattr-ff573d417a6c2284.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/libfiletime-0be64287acb654c5.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/liblibc-527eb54b3bbce9f8.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/libflate2-f11a11510cdd5b7c.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/libminiz_oxide-fe189d1972879e03.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/libadler32-3c76c2dfcd9290a7.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/libcfg_if-ee99338ab4f485d5.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/libcrc32fast-6e5702c9e4f46fde.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/libcfg_if-a1f8b378dc0c2436.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/libtoml-05606085cc9a8eb4.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/libserde-0919d4e5d80ffd23.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/libwalkdir-e72d8fcb7aada1ed.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/libsame_file-8d9eb47898aba420.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/libitertools-01e7cd9951bb31ef.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/libeither-fa8d2728fb88c1d1.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/libregex-4116ec66810208ad.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/libthread_local-d4006d4980ce5404.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/libonce_cell-f685a49b706ce2b8.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/libregex_syntax-0467555b2c4dac1b.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/libaho_corasick-c2dfd7c98e633f5b.rlib" "/private/tmp/nix-build-elan-0.11.0.drv-0/source/target/x86_64-apple-darwin/release/deps/libmemchr-a8a237840e4b38fe.rlib" "/nix/store/akm781c35gr7mb14g5h0kqlk3cvyzmij-rustc-1.51.0/lib/rustlib/x86_64-apple-darwin/lib/libstd-0663ee2c1c537ca4.rlib" "/nix/store/akm781c35gr7mb14g5h0kqlk3cvyzmij-rustc-1.51.0/lib/rustlib/x86_64-apple-darwin/lib/libpanic_unwind-51315f7bcfa049eb.rlib" "/nix/store/akm781c35gr7mb14g5h0kqlk3cvyzmij-rustc-1.51.0/lib/rustlib/x86_64-apple-darwin/lib/libobject-ca0009034960ab56.rlib" "/nix/store/akm781c35gr7mb14g5h0kqlk3cvyzmij-rustc-1.51.0/lib/rustlib/x86_64-apple-darwin/lib/libaddr2line-64ee72358536cb5c.rlib" "/nix/store/akm781c35gr7mb14g5h0kqlk3cvyzmij-rustc-1.51.0/lib/rustlib/x86_64-apple-darwin/lib/libgimli-37ffdd8ede97efc7.rlib" "/nix/store/akm781c35gr7mb14g5h0kqlk3cvyzmij-rustc-1.51.0/lib/rustlib/x86_64-apple-darwin/lib/librustc_demangle-483f19d8e4da8768.rlib" "/nix/store/akm781c35gr7mb14g5h0kqlk3cvyzmij-rustc-1.51.0/lib/rustlib/x86_64-apple-darwin/lib/libhashbrown-dc4b303f88932edd.rlib" "/nix/store/akm781c35gr7mb14g5h0kqlk3cvyzmij-rustc-1.51.0/lib/rustlib/x86_64-apple-darwin/lib/librustc_std_workspace_alloc-92d38392fba41e6a.rlib" "/nix/store/akm781c35gr7mb14g5h0kqlk3cvyzmij-rustc-1.51.0/lib/rustlib/x86_64-apple-darwin/lib/libunwind-e5d8068e42c74666.rlib" "/nix/store/akm781c35gr7mb14g5h0kqlk3cvyzmij-rustc-1.51.0/lib/rustlib/x86_64-apple-darwin/lib/libcfg_if-22c964ee3d0265a5.rlib" "/nix/store/akm781c35gr7mb14g5h0kqlk3cvyzmij-rustc-1.51.0/lib/rustlib/x86_64-apple-darwin/lib/liblibc-0a3a3c5b22954ea6.rlib" "/nix/store/akm781c35gr7mb14g5h0kqlk3cvyzmij-rustc-1.51.0/lib/rustlib/x86_64-apple-darwin/lib/liballoc-0ddbdbad7d99e35c.rlib" "/nix/store/akm781c35gr7mb14g5h0kqlk3cvyzmij-rustc-1.51.0/lib/rustlib/x86_64-apple-darwin/lib/librustc_std_workspace_core-f6456a5a8a2c2723.rlib" "/nix/store/akm781c35gr7mb14g5h0kqlk3cvyzmij-rustc-1.51.0/lib/rustlib/x86_64-apple-darwin/lib/libcore-6df2a021d0eda86a.rlib" "/nix/store/akm781c35gr7mb14g5h0kqlk3cvyzmij-rustc-1.51.0/lib/rustlib/x86_64-apple-darwin/lib/libcompiler_builtins-94ac56476cd78265.rlib" "-lcurl" "-liconv" "-lSystem" "-lresolv" "-lc" "-lm"
  = note: ld: library not found for -liconv
          clang-7: error: linker command failed with exit code 1 (use -v to see invocation)
```
Complete log available here: https://gist.github.com/samuela/e4bad268c5ae02028ad7a3cf5d147cc3.

**To Reproduce**
1. On a x86_64-darwin machine, run `nix-env -iA nixpkgs.elan`.

**Expected behavior**
The build to succeed.

**Screenshots**

**Additional context**

**Notify maintainers**

@Kha @gebner 

**Metadata**
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.
```
❯ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-darwin"`
 - host os: `Darwin 19.6.0, macOS 10.15.7`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.4pre19700101_c235bed`
 - channels(root): `"nixpkgs-21.05pre283153.7919518f023"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixpkgs`
```

Maintainer information:
```yaml
# a list of nixpkgs attributes affected by the problem
attribute: elan
# a list of nixos modules affected by the problem
module:
```

**Comment by samuela** [comment]
Just tried with latest from nixpkgs-unstable and it works fine, so I guess I'll close for now.

**Comment by gebner** [comment]
I believe this was fixed in #123081
